fe/fe-core/src/main/java/com/starrocks/http/rest/ShowMetaInfoAction.java

```diff
@@ -97,7 +99,10 @@ public void execute(BaseRequest request, BaseResponse response) {
 
         switch (Action.getAction(action.toUpperCase())) {
             case SHOW_DB_SIZE:
-                response.getContent().append(gson.toJson(getDataSize()));
+                response.getContent().append(gson.toJson(getDataSize(true)));
```

Comment:
possibly, we share expose this as logic size and physical size, usually
- logic size: single replica size
- physical size: sum up all replica size

Reply:
not intend to break backward compatibility for this rest api, so just wanna adding a new Action to indicate retrieving of data size for all replicas.